packages/@tailwindcss-upgrade/src/codemods/migrate-at-config.ts

```diff
@@ -39,7 +38,7 @@ export function migrateAtConfig(
     let sheetPath = sheet.file
     let configPath = configFilePath
 
-    let relativePath = path.relative(path.dirname(sheetPath), configPath)
+    let relativePath = path.relative(path.dirname(sheetPath), configPath).replaceAll('\\', '/')
```

Comment:
I don't completely recall but we have a normalizePath function in `@tailwindcss/node` and I _think_ it can handle relative paths. Might be good to use that here.